src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -310,4 +307,20 @@ public void setNotFailingStatuses(Set<Status> notFailingStatuses) {
             this.notFailingStatuses = notFailingStatuses;
         }
     }
+
+    public void setQualifier(@NonNull String jsonFile, @NonNull String qualifier) {
```

Comment:
can you update about it  README.md and LiveDemoTest - very basic sample is good enough

Reply:
@damianszczepanik Should I amend existing LiveDemoTest or add an additional one?  Similarly with README.md - amend existing example or add another one?

Reply:
amend :) both!

Reply:
Done